contracts/ServiceRegistry.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.4.17;
+
+import "./ManagerProxyTarget.sol";
+
+
+/**
+ * @title ServiceRegistry
+ * @dev Maintains a registry of service metadata associated with service provider addresses (transcoders/orchestrators)
+ */
+contract ServiceRegistry is ManagerProxyTarget {
+    // Store service metadata
+    struct Record {
```

Comment:
If we changed the Record struct in the future, the storage layout will be screwy right?  If that's the case, I prefer to have the mapping from `address` to `string`, that way this potential mistake doesn't happen.

Reply:
Adding additional fields to the `Record` struct in an upgrade is safe as long as 1) we do not remove/replace existing fields 2) the `Record` struct is not used in an array (it is only used in a mapping right now)